src/components/shared/prompt/prompt.styles.ts

```diff
@@ -58,28 +58,39 @@ export const ButtonContainer = styled.div<SizeProps>`
 	display: flex;
 	flex-direction: column;
 	justify-content: center;
-	padding: 0 1.5rem 4rem;
+	padding: 0 ${Spacing["spacing-24"]} ${Spacing["spacing-64"]};
 
 	${MediaQuery.MinWidth.md} {
 		align-items: center;
-		padding: ${(props) => (props.size === "large" ? "2rem 4rem 4rem" : "2.5rem 1.5rem 2rem")};
 		flex-direction: row-reverse;
+
+		${(props) =>
+			css`
+				padding: ${props.size === "large"
+					? `${Spacing["spacing-32"]} ${Spacing["spacing-64"]} ${Spacing["spacing-64"]}`
+					: `${Spacing["spacing-40"]} ${Spacing["spacing-24"]} ${Spacing["spacing-32"]}`};
+			`}
 	}
 `;
 
 export const LabelContainer = styled.div<SizeProps>`
 	display: flex;
 	flex-direction: column;
 	text-align: center;
-	padding: 4rem 1.5rem 1.5rem;
+	padding: ${Spacing["spacing-64"]} ${Spacing["spacing-24"]} ${Spacing["spacing-24"]};
 
 	${MediaQuery.MinWidth.md} {
-		padding: ${(props) => (props.size === "large" ? "4rem 4rem 0rem 4rem" : "2rem 1.5rem 0")};
+		${(props) =>
+			css`
+				padding: ${props.size === "large"
+					? `${Spacing["spacing-64"]} ${Spacing["spacing-64"]} 0rem ${Spacing["spacing-64"]}`
+					: `${Spacing["spacing-32"]} ${Spacing["spacing-24"]} 0`};
+			`}
```

Comment:
the `css` should be applied on the nested string literal inside of a function

replace this with

```
padding: ${(props) =>
			props.size === "large"
				? css`
						${Spacing["spacing-64"]} ${Spacing["spacing-64"]} 0rem ${Spacing["spacing-64"]}
				  `
				: css`
						${Spacing["spacing-32"]} ${Spacing["spacing-24"]} 0
				  `};
```

update for `ButtonContainer` as well